graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -998,12 +1034,12 @@
     "node_size = 120\n",
     "min_c = 0.3\n",
     "num_graphs = len(raw_graphs)\n",
-    "targets = utils_np.graphs_tuple_to_data_dicts(test_values[\"target\"])\n",
+    "targets = graphs_touple_to_graph_list(targets_ge)\n",
```

Comment:
I have checked and this one, and the one below just works with:
`utils_np.graphs_tuple_to_data_dicts`

so you can remove the `graphs_touple_to_graph_list` method.

(You will need to also revert the changes in line 1017 and line 1043)